applications/graphql-lambda/dynamodb-policy.json.tpl

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Assuming this is the standard copied from AWS grants permissions groups console?

Reply:
[Yup](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_examples_dynamodb_specific-table.html).